setup/deb-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+echo 'Setting up your developmental environment. This may take a while.'
+
+echo '[*] Adding keys...'
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+
+echo '[*] Installing GNUpg...'
+sudo apt-get install gnupg
+
+echo '[*] Adding Keys for rvm...'
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB && echo '[*] Installing rvm...' && \curl -sSL https://get.rvm.io | bash -s stable
+
+echo '[*] Installing Ruby-2.5.0...'
+sudo rvm install ruby-2.5.0
+
+echo '[*] Install node.js...'
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+
+echo '[*] Installing bundler...'
+gem install bundler
+
+echo '[*] Installing Gems and dependencies...'
+sudo apt-get install libmysqlclient-dev libpq-dev libqtwebkit-dev
+bundle install
+
+echo '[*] Installing yarn...'
+sudo apt-get install yarn
+
+echo '[*] Installing node_modules...'
+yarn
+
+echo '[*] Installing phantomjs-prebuilt...'
+sudo yarn global add phantomjs-prebuilt
+
+echo '[*] Installing bower...'
+sudo yarn global add bower
+
+echo '[*] Installing bower modules...'
+bower install
+
+echo '[*] Installing pandoc...'
+sudo apt-get install pandoc
+
+echo '[*] Creating configuration files...'
+cp config/application.example.yml config/application.yml
+cp config/database.example.yml config/database.yml
```

Comment:
I think what we want to do here is to change the example database.yml file to use a different user instead of root, and then create that user in mysql with the password specified in the file and grant ownership to that user for the new databases.

Otherwise, this will break at the rake step unless the root mysql password has been set to the same one in database.yml.